### DISCUSSION TITLE
Any way to remove the skipping of installation of a plugin

### DISCUSSION THREAD

**Discussion by fuzails** [ionic-team/capacitor#3431] [Q&A]
Hi

I get the following

Found 3 incompatible Cordova plugins for ios, skipped install, which is fine but there is a version of one of the cordova plugins that has ios compatibility now in beta format. As such I would like to disable skipping that plugin to test if it works as expected.

Specifically cordova-plugin-googlemaps

**Comment by jcesarmobile** [comment]
It's not really possible unless the plugin author changes the plugin id or we remove it from the skip list

I've sent a PR to remove it from the skip list https://github.com/ionic-team/capacitor/pull/3436
